patches/PB-319__fix_pagebuilder_module__2.3.1.patch

```diff
@@ -0,0 +1,1358 @@
+diff -Nuar a/vendor/magento/module-page-builder/Controller/Adminhtml/Stage/Preview.php b/vendor/magento/module-page-builder/Controller/Adminhtml/Stage/Preview.php
+new file mode 100644
+index 000000000..be64e560d
```

Comment:
This is not critical but it would be nice to remove indexes from new patches, we did this in all patches.